templates/usr/local/kafka/config/server.properties.j2

```diff
@@ -58,21 +61,42 @@ background.threads={{ server.background_threads }}
 queued.max.requests={{ server.queued_max_requests }}
 {% endif %}
 
+{% if server.listeners %}
+# default: null
+# Listener List - Comma-separated list of URIs we will listen on and their protocols. Specify hostname as 0.0.0.0 to bind to all interfaces. Leave hostname empty to bind to default interface. Examples of legal listener lists: PLAINTEXT://myhost:9092,TRACE://:9091 PLAINTEXT://0.0.0.0:9092, TRACE://localhost:9093
+listeners={{ server.listeners }}
+{% endif %}
+
+{% if server.advertised_listeners %}
+# default: null
+# Listeners to publish to ZooKeeper for clients to use, if different than the listeners above. In IaaS environments, this may need to be different from the interface to which the broker binds. If this is not set, the value for `listeners` will be used.
+advertised_listeners={{ server.advertised_listeners }}
```

Comment:
I was following the same naming convention for example `num.io.threads` as `server.num_io_threads` and other variables int this file.